public/css/style.css

```diff
@@ -257,10 +284,34 @@
     color: #1dc3e1;
 }
 
+.tag-list {
+    display:inline;
+    padding: 0;
+}
+
+.tag-list a {
+    color: black;
+    text-decoration: none;
+}
+
+.tag-list a:hover {
+    color: #d50032;
+}
+
+.tag-list a:visited {
+    color: #1dc3e1;
+}
+
 .bg-primary, .btn-primary {
     background-color: #d50032 !important;
 }
 
+.btn-align {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+}
+
```

Comment:
To clarify, what does flex do for this button, i can't see it doing anything differently from block.